ui/src/index.html

```diff
@@ -21,6 +21,17 @@
         google.charts.load('current', {'packages': ['corechart']});
     </script>
 
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-112406425-1"></script>
+   <script>
+       window.dataLayer = window.dataLayer || [];
```

Comment:
Nit: Spacing. (This is indented in four spaces from the script tag, I believe we are doing two? I realized we didn't do that for the charts, I'll go back and fix that as a follow up)

Reply:
Changed to two in both places.